archey/entry.py

```diff
@@ -0,0 +1,22 @@
+"""Entry base class"""
+
+from abc import ABC as AbstractBaseClass, abstractmethod
+
+from archey.configuration import Configuration
+
+
+class Entry(AbstractBaseClass):
```

Comment:
I've thought setting this base class as `Abstract` would be good idea, waiting for your thoughts about this.

Reply:
Yeah, that's good, especially since it stops the possibility of instantiating an entry using `Entry()` directly.